### DISCUSSION TITLE
If possible implement flux or SD3.5 in foocus

### DISCUSSION THREAD

**Discussion by ApenasNiko** [lllyasviel/Fooocus#3748] [Ideas]
Technology advances rapidly, and although the SDXL represented an important milestone in imaging, it is already showing signs of obsolescence. Limitations in detailed processing and over-reliance on manual adjustments to prompts indicate that it is time to adopt more robust, modern solutions.
Flux, with its dynamic architecture and contextual learning integration, enables significantly more realistic and adaptive imaging, eliminating the need for repetitive adjustments. SD3.5 brings significant improvements in visual coherence and the ability to interpret complex prompts, which is essential for more efficient and professional workflows.
While SDXL still delivers quality, it no longer meets the demands of creators looking for precision, flexibility and innovation. The community needs tools that keep up with the growing demands for hyper-realistic details and streamlined processes.
Implementing Flux or SD3.5 now is not just a matter of upgrading; it is an investment in the creative evolution and potential of the platform. Let's take this step together to reach a new level in quality and productivity!

**Comment by IPv6** [comment]
> Let's take this step together to reach a new level in quality and productivity!

Yes, speaking of "take this step together" - you are free to re-write Fooocus internals with full support of new architectures yourself. 
And send a pull request :) It will be tested together then.

Or better use Forge/SimpleSDXL/etc, where Flux support already implemented

**Reply by poor7** [reply]
SimpleSDXL made in such a way that it cannot work with Сolab. :/

**Reply by IPv6** [reply]
Are you speaking about free tier? Flux/SD3.5 will never work in Colab normally above simple text to image in that case, colab GPUs (free tier) are too limited for them, this networks are very demanding //

**Comment by Yogesh-DevHub** [comment]
Ruined fooocus should work on colab with flux model: https://youtu.be/3nMo_rLD02g?si=1rvQUx9Qlm5ZqBxp
